pom.xml

```diff
@@ -492,6 +492,11 @@
         <version>${guava.version}</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
+        <groupId>com.github.ben-manes.caffeine</groupId>
+        <artifactId>caffeine</artifactId>
+        <version>2.9.1</version>
```

Comment:
nit: maybe define `caffeine.version` and put it under `properties`.

Reply:
done